File: abc/abc167/a.py
# https://atcoder.jp/contests/abc167/submissions/34773622

s = input()
t = input()
cnt = 0
ans = True
while cnt < len(s):
  if s[cnt] != t[cnt]: ans = False
  cnt += 1
print("Yes" if ans == True else "No")

File: abc/abc167/a.cs
// https://atcoder.jp/contests/abc167/submissions/46443598

using System;

class A
{
  static void Main()
  {
    string s = Console.ReadLine();
    string t = Console.ReadLine();

    bool ans = true;
    for(int i = 0; i < s.Length; i++)
    {
      if(s[i] != t[i])
      {
        ans = false;
        break;
      }
    }

    if(ans)
    {
      Console.WriteLine("Yes");
    }
    else
    {
      Console.WriteLine("No");
    }
  }
}

